clouddriver-elasticsearch/src/main/java/com/netflix/spinnaker/clouddriver/elasticsearch/ops/BulkUpsertEntityTagsAtomicOperation.java

```diff
@@ -246,8 +246,16 @@ private static void addTagMetadata(Date now, EntityTags entityTags) {
     entityTags.getTags().forEach(tag -> entityTags.putEntityTagMetadata(tagMetadata(tag.getName(), now)));
   }
 
-  private static AccountCredentials lookupAccountCredentials(AccountCredentialsProvider accountCredentialsProvider,
-                                                             String entityRefAccountId) {
+  private static AccountCredentials lookupAccountCredentialsByAccountName(AccountCredentialsProvider accountCredentialsProvider,
```

Comment:
I dig it ... 

I'd also be ok with

```
private static AccountCredentials lookupAccountCredentials(AccountCredentialsProvider accountCredentialsProvider,
                                                             String entityRefAccountIdOrName) {
    return accountCredentialsProvider.getAll().stream()
      .filter(c -> entityRefAccountIdOrName.equals(c.getAccountId()) || entityRefAccountIdOrName.equals(c.getName()))
      .findFirst()
      .orElseThrow(() -> new IllegalArgumentException("No credentials found for account '" + entityRefAccountIdOrName + "'"));
  }
```

Just one method that takes a param representing either an account name or id and does the match ... slightly less duplicated code. It's a `private` method so the context is all kept within this class anyhow.

Otherwise LGTM